### PR TITLE
build: split deploy and build scripts

### PR DESCRIPTION
**Pull request by CerDennis**
Refs:SDKUI-14

### CODE REVIEW

**Review by beawar — COMMENTED** [review]
I'm not sure duplicating the deploy script to the install, just to include the build, is convenient for the future. It would mean keeping in mind that all changes to the deploy should be replicated also to the install.
I'm ok with the new command install, but I think it should invoke the build and the deploy scripts directly, without code duplication.

Also, I think these changes represent a feat, and are borderline with a breaking change too.

**Comment by CerDennis** [comment]
> I'm not sure duplicating the deploy script to the install, just to include the build, is convenient for the future. It would mean keeping in mind that all changes to the deploy should be replicated also to the install. I'm ok with the new command install, but I think it should invoke the build and the deploy scripts directly, without code duplication.
> 
> Also, I think these changes represent a feat, and are borderline with a breaking change too.

Sure I'm removing the redundant code, about the breaking change as you said it's not breaking but it's changing the behavior so if you prefer the breaking change it's not a problem for me or we can toss a coin

**Comment by beawar** [comment]
>  about the breaking change as you said it's not breaking but it's changing the behavior so if you prefer the breaking change it's not a problem for me or we can toss a coin

Up to you as maintainers deciding it, I just wrote my 2 cents :wink:

File: scripts/deploy.js [hunk 1]
```diff
@@ -6,10 +6,12 @@
 
 /* eslint-disable no-console */
 const chalkTemplate = require('chalk');
-const { handler: build, builder: buildOptions } = require('./build');
+const { builder: buildOptions } = require('./build');
```
Comment (beawar) [inline scripts/deploy.js]: The deploy script is still receiving the build options, but they are not used anymore

File: scripts/install.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      desc: 'Localhost port to use',
+      alias: 'p',
+      default: '',
+    },
+  }
+);
+
+exports.handler = async (options) => {
+  await build(options);
+  await deploy(options);
```
Comment (beawar) [inline scripts/install.js]: Both the scripts are receiving the options, but the options declared in the script are just the ones of the deploy script

**Review by beawar — APPROVED** [review]

**Review by yukiLu6 — APPROVED** [review]

**Pull request merged**